README.md

```diff
@@ -41,6 +41,55 @@ Be sure to also insert the [Oauth table](config/oauth_dev_dump.sql) so that you
 
 
 
+
+
+## Docker
+
+The Docker container requires that either the port for the Postgres instance is forward or that the host is set to an exact IP or domain.
+
+Using Docker:
+
+```bash
+# enter the project directory
+cd density
+
+# builds a Docker image
+#   -t dictates that the image is tagged as 'density'
+docker build -t density .
```

Comment:
will fix on next push. idk how to alter the installation easily enough to not require this since the `vagrant` user has sudo while running the provisioning script.
i'll try to better understand unix permissions and fix this.